awscli/customizations/eks/update_kubeconfig.py

```diff
@@ -230,12 +237,13 @@ def _expand_path(self, path):
 
 
 class EKSClient(object):
-    def __init__(self, session, cluster_name, role_arn, parsed_globals=None):
+    def __init__(self, session, cluster_name, role_arn, proxy_url, parsed_globals=None):
```

Comment:
To maintain backwards compatibility for any CLI customers that have custom code utilizing this class, let's make `proxy_url` an optional argument.
```suggestion
    def __init__(self, session, cluster_name, role_arn, proxy_url=None, parsed_globals=None):
```